proto/lbm/foundation/v1/foundation.proto

```diff
@@ -107,6 +111,14 @@ message DecisionPolicyWindows {
   google.protobuf.Duration min_execution_period = 2 [(gogoproto.stdduration) = true, (gogoproto.nullable) = false];
 }
 
+// OutsourcingDecisionPolicy is a dummy decision policy which is set after
+// the operator has been updated into x/group's group policy.
```

Comment:
Does comment mean that the codes(OutsourcingDecisionPolicy) is from x/group in cosmos-sdk?

Reply:
No, there is no corresponding code in `x/group`. It would be set if we don't need to use `x/foundation`'s proposal feature anymore.